app/controllers/billing_expenses_controller.rb

```diff
@@ -0,0 +1,59 @@
+class BillingExpensesController < ApplicationController
+  before_action :set_billing_expense, only: [:show, :edit, :update, :destroy]
+  before_action :set_volunteer
+
+  def index
+    billing_expense = BillingExpense.new(volunteer: @volunteer)
+    authorize billing_expense
```

Comment:
You don't need to create a new billing expense Instance, you can just pass the Class to it:
```ruby
authorize BillingExpense
```

Reply:
Yes, it was forgotten there because initially the policy allowed the volunteer to file billing expenses.